pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -659,3 +659,15 @@ def test_getitem_non_found_tuple():
     )
     with pytest.raises(KeyError, match=r"\(2\.0, 2\.0, 3\.0\)"):
         df.loc[(2.0, 2.0, 3.0)]
+
+
+def test_loc_multiindex_columns_one_level():
```

Comment:
deep down this is a `DataFrame.__getitem__` test

Reply:
Thx, moved. Thought about were to put it, but looking at the place were the modification was necessary should have answered it easily